packages/asc-ui/src/components/Header/Header.stories.tsx

```diff
@@ -2,76 +2,75 @@ import * as React from 'react'
 import { storiesOf } from '@storybook/react'
 import CenteredElement from '../../internals/CenteredElement/CenteredElement'
 import BoxWrapper from '../../internals/Box/BoxWrapper'
-import { ThemeProvider } from '../../index'
-import HeaderShort from './HeaderShort'
+import Header from './Header'
 import ContentFiller from '../../internals/ContentFiller/ContentFiller'
-import HeaderTall from './HeaderTall'
+import HeaderSearch from './HeaderSearch'
+import HeaderMenu from './HeaderMenu'
+import HeaderContent from './HeaderContent'
 
 const outsideBackgoundColor = '#E6E6E6'
 const contentBackgrountColor = '#ffffff'
 
 const HeaderShortStory: React.FC<{}> = () => (
-  <ThemeProvider themeName="default">
-    <>
-      <BoxWrapper backgroundColor={outsideBackgoundColor}>
-        <HeaderShort
-          title="Data en informatie"
-          homeLink="http://data.amsterdam.nl"
-          search={<CenteredElement> Search </CenteredElement>}
-          menu={<CenteredElement> M </CenteredElement>}
-          headerSize="short"
-        />
-        <ContentFiller backgroundColor={contentBackgrountColor} />
-      </BoxWrapper>
-    </>
-  </ThemeProvider>
+  <BoxWrapper backgroundColor={outsideBackgoundColor}>
```

Comment:
why is the theme provider removed?

Reply:
Because we already use it via .storybook/config.js, so we won't need to include them all the time :)

Reply:
NICE!